.github/workflows/deploy.yaml

```diff
@@ -80,3 +80,32 @@ jobs:
           --timeout 15m \
           ${{matrix.stage}}-energy-comparison-table \
           energy-comparison-table/infrastructure/app/chart/energy-comparison-table
+  slack-success:
+    name: Slack Notification (Success)
+    runs-on: ubuntu-22.04
```

Comment:
Can this use `runs-on: runs-on` like our other jobs or does it need to be a built-in runner?

I see the [example uses Ubuntu](https://github.com/citizensadvice/message-slack-action?tab=readme-ov-file#example-usage), but it doesn't say why.

Reply:
Good question! 

Looks like all of the workflows on `energy-comparison-table` use `runs-on: ubuntu-22.04`, but `smart-meter-tool`, for example, uses `runs-on:runs-on`. 

I'd assumed that because the rest of the `deploy` workflow uses `runs-on: ubuntu-22.04` these new jobs should use the same runner (i.e. use the same runner throughout a single workflow). I'll check whether that assumption was correct, though.